.gitignore

```diff
@@ -16,4 +16,5 @@
 .direnv
 
 **/package-lock.json
+**/yarn.lock
```

Comment:
yarn.lock should remain in git. https://classic.yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control